Frameworks/include/CFCppBase.h

```diff
@@ -0,0 +1,90 @@
+//******************************************************************************
+//
+// Copyright (c) 2016 Microsoft Corporation. All rights reserved.
```

Comment:
There's quite a lot of files out there with an outdated header. Should we just do a mass replace? It might clear up a lot of this.

---
In reply to: [83494095](https://github.com/Microsoft/WinObjC/pull/1153#discussion_r83494095) [](ancestors = 83494095)